amical/data_processing.py

```diff
@@ -613,6 +621,25 @@ def select_clean_data(
         verbose=verbose,
     )
 
+    if check_clean_params and display:
```

Comment:
So this is the only place where `check_clean_params` is used, but in half of the cases (when `display is False`), it has no effect. This is confusing and also not in line with the docstring you wrote, so I'd suggest making this check only care about one boolean rather than 2. Maybe `display`, which is right here, is already the natural choice, and we don't actually need a new keyword argument for this ?

Reply:
Right, at first I was using only `display`, and I agree it is simpler. The only thing is that this would duplicate the plots in any legacy code that uses `show_clean_params()` already (outside of `select_clean_data()`) and that has `display=True`. In any case, I agree that it would be more intuitive to only have `display=True`.  I will revert to that.

Reply:
I see. Well this is unfortunate but I don't see any better approach, and I'd err on the side of not sacrifying clarity of the long term API only to avoid a minor side effect for early adopters. We should emphasise it in the next version's release notes though. I'll add a note for this !

Reply:
Sounds good, thank you!